src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -134,6 +157,28 @@ class adminLogs extends viewModelBase {
                 }
             }
         });
+
+        this.startDateToUse = ko.pureComputed(() => {
+            return this.useMinStartDate() ? null : this.startDate().utc().format(generalUtils.utcFullDateFormat);
+        });
+
+        this.endDateToUse = ko.pureComputed(() => {
+            return this.useMaxEndDate() ? null : this.endDate().utc().format(generalUtils.utcFullDateFormat);
+        });
+    }
+
+    dateFormattedAsUtc(localDate: moment.Moment) {
+        return ko.pureComputed(() => {
```

Comment:
I think we don't need to return pure computed here.

Reply:
done